opencga-app/app/scripts/docker/opencga-init/Dockerfile

```diff
@@ -56,4 +56,4 @@ RUN chmod +x /tmp/setup.sh
 # It is the responsibility of the setup.sh
 # script to initialise the volume correctly
 # and apply any runtime config transforms.
-ENTRYPOINT ["/tmp/setup.sh"]
\ No newline at end of file
+ENTRYPOINT [ "/bin/bash","/tmp/setup.sh" ]

```

Comment:
Is this needed?

Reply:
So as entrypoint was "/bin/bash" - so when key was added its trying to execute the key.

Could have added the script name into the ARM template JSON or do this...

Reply:
Could this be changed from ENTRYPOINT [...] to CMD ["/tmp/setup.sh"] for the same effect?

Reply:
Issue is then when override the command end up with ```/bin/bash secretkey```

Reply:
Ah gotcha